.env.dev

```diff
@@ -6,6 +6,8 @@ REACT_APP_ENV=development
 REACT_APP_SUPPORT_SERVICE="Telegram"
 REACT_APP_SUPPORT_URL="https://cere.network/discord"
 
+REACT_APP_WALLET_CONNECT_V2_TOKEN="1c577e873c4c74d6a6a870a8c8128232"

```

Comment:
How is the token used? If it's a kind of auth token we shouldn't store it in the source code.